test/test_ops.py

```diff
@@ -0,0 +1,206 @@
+import collections
+import numbers
+from typing import Callable
+
+import torch
+from torch.testing._internal.common_utils import \
+    (TestCase, run_tests)
+from torch.testing._internal.common_methods_invocations import \
+    (SampleInput, op_db, SkipInfo)
+from torch.testing._internal.common_device_type import \
+    (instantiate_device_type_tests, ops)
+
+# Skips which are frequently used.
+reference_eager_all_skip = SkipInfo(
+    "TestOpInfo", "test_reference_eager", device_type='xla')
+
+reference_eager_int_skip = SkipInfo(
+    "TestOpInfo",
+    "test_reference_eager",
+    device_type='xla',
+    dtypes=(torch.int64,))
+
+reference_eager_float_skip = SkipInfo(
+    "TestOpInfo",
+    "test_reference_eager",
+    device_type='xla',
+    dtypes=(torch.float,))
+
+SkipEntry = collections.namedtuple(
+    "SkipEntry", ["op_name", "variant_name", "skip"], defaults=["", "", ()])
+
+
+def update_skips():
```

Comment:
Kept skips here for now but should they be moved to `pytorch_test_base.py`?

Reply:
Though a problem with keeping skips here is that if a new `OpInfo` is added, the developer might have to add an entry here :(

They can add the `Skip` in `common_method_invocations` (in the core) as well but then that would mean that the skips won't be at one place.